avaframe/runUpdateBenchmarkTestsCom1DFA.py

```diff
@@ -0,0 +1,143 @@
+from avaframe.log2Report import generateReport as gR
+from avaframe.log2Report import generateCompareReport
+from avaframe.ana3AIMEC import ana3AIMEC, dfa2Aimec, aimecTools
+from avaframe.out3Plot import outQuickPlot
+from avaframe.out1Peak import outPlotAllPeak as oP
+from avaframe.in3Utils import fileHandlerUtils as fU
+from avaframe.in3Utils import initializeProject as initProj
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+from benchmarks import simParametersDict
+
+
+# +++++++++REQUIRED+++++++++++++
+# Which result types do we want to save in the benchmarks
+outputVariable = ['ppr', 'pfd', 'pfv']
+# ++++++++++++++++++++++++++++++
+
+# log file name; leave empty to use default runLog.log
+logName = 'runUpdateBenchmarkTestsCom1DFA'
+
+# Load settings from general configuration file
+cfgMain = cfgUtils.getGeneralConfig()
+
+# load all benchmark info as dictionaries from description files
+testDictList = tU.readAllBenchmarkDesDicts(info=False)
+
+# filter benchmarks to extract only desiered ones
+type = 'TAGS'
+valuesList = ['standardTest']
+testList = tU.filterBenchmarks(testDictList, type, valuesList, condition='and')
+
+# Set directory for full standard test report
+outDir = pathlib.Path('..', 'benchmarks')
+fU.makeADir(outDir)
+
+log = logUtils.initiateLogger(outDir, logName)
+log.info('The following benchmark tests will be updated ')
+for test in testList:
+    log.info('%s' % test['NAME'])
+
+# run Standard Tests sequentially
+for test in testList:
+
+    avaDir = test['AVADIR']
+
+    # ################################
+    # Fetch benchmark test info
+    # toDo: this needs to be changed. We want to read this from the json file
+    # once we updated the benchmarks, we can remove this and remove the fetchBenchParameters function
+    benchDict = simParametersDict.fetchBenchParameters(test['NAME'])
+    test['Test Info'] = benchDict['Test Info']
+    test['simType'] = benchDict['simType']
+    # #####################################
+
+    simNameRef = test['simNameRef']
+    refDir = pathlib.Path('..', 'benchmarks', test['NAME'])
+
+    # Clean input directory(ies) of old work and output files
+    initProj.cleanSingleAvaDir(avaDir, keep=logName)
+
+    # RunCom1DFA
+    # Load input parameters from configuration file for standard tests
+    avaName = pathlib.Path(avaDir).name
+    standardCfg = refDir / ('%s_com1DFACfg.ini' % test['AVANAME'])
+    modName = 'com1DFA'
+
+    # change the ini file to force particle initialization
+    updater = ConfigUpdater()
+    updater.read(standardCfg)
+    updater['GENERAL']['initialiseParticlesFromFile'].value = 'False'
+    updater.update_file()
+    # Set timing
+    startTime = time.time()
+    # call com1DFA run
+    dem, plotDict, reportDictList, simDF = com1DFA.com1DFAMain(avaDir, cfgMain, cfgFile=standardCfg)
+    endTime = time.time()
+    timeNeeded = endTime - startTime
+    log.info(('Took %s seconds to calculate.' % (timeNeeded)))
```

Comment:
Mh, I think this could also be helpful information: collect all calculation times in a dict or similar and write it out at the end. Maybe even to a file. This could be used to indicate if something is wrong. I.e. suddenly taking a lot longer. Of course, this varies by machine but could be used during a github action workflow...